src/Libraries/GeometryUI/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,79 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using Dynamo.Utilities;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GeometryUI
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryCondition")]
+    [NodeCategory("Geometry.PanelSurface")]
+    [NodeDescription("PanelSurfaceBoundaryConditionDropDownDesc", typeof(Properties.Resources))]
+    [OutPortNames(">")]
+    [OutPortTypes(nameof(PanelSurfaceBoundaryCondition))]
+    [OutPortDescriptions("PanelSurface BoundaryCondition enum value")]
+    [IsDesignScriptCompatible]
+    public class PanelSurfaceBoundaryConditionDropDown : DSDropDownBase
```

Comment:
I think there is an enum based dropdown...
https://github.com/DynamoDS/DynamoRevit/blob/bcb122017c2e5ebe63e4198f8b8f84a3abc5305a/src/Libraries/RevitNodesUI/GenericClasses.cs#L128